### DISCUSSION TITLE
Error on connecting to custom domain

### DISCUSSION THREAD

**Discussion by patrickhars** [community/community#23436] [Other Feature Feedback, Questions, & Ideas]
<p>Hi, this is my first time making a website and so I do not know what to do now that there is this error. I wanted to connect my godaddy domain greentree.group to my github page <a href="http://patrickhars.github.io/GREENTREE" rel="noopener nofollow ugc">patrickhars.github.io/GREENTREE</a> but it doesnt seem to be working. I have also set up the forwarding to <a href="https://patrickhars.github.io/GREENTREE" rel="noopener nofollow ugc">https://patrickhars.github.io/GREENTREE</a> but when I open greentree.group it redirects to 404 error while if I enter greentree.group/GREENTREE it works. Also, sometimes I get a privacy error for some reason.<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/f/3/f38ea05c9572aa48975ccaf409375b0738086e65.png" data-download-href="/uploads/short-url/yKBBgeU7fxs89i3FyYmP1huFeF7.png?dl=1" title="1" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/85603170/181076911-7f3126e5-96c5-471c-b7c7-b349641982d4.png" alt="1" data-base62-sha1="yKBBgeU7fxs89i3FyYmP1huFeF7" width="689" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/f/3/f38ea05c9572aa48975ccaf409375b0738086e65_2_689x500.png, https://aws1.discourse-cdn.com/github/optimized/3X/f/3/f38ea05c9572aa48975ccaf409375b0738086e65_2_1033x750.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/f/3/f38ea05c9572aa48975ccaf409375b0738086e65_2_1378x1000.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/f/3/f38ea05c9572aa48975ccaf409375b0738086e65_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">1</span><span class="informations">1653×1199 64.8 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by otter-computer** [comment]
<p>Hi <a class="mention" href="/patrickhars">@patrickhars</a>, welcome to the GitHub Support Community! It looks like you’ve got an extra <code>A</code> record pointing to <code>184.168.131.241</code> that may be interfering with routing to your domain. You’ll need to remove that record before your domain points completely towards GitHub.</p>
<p>If you’re not able to remove this from your DNS settings it may be added by your DNS provider automatically. You might need to turn off “domain parking”, forwarding, or another service they provide, or contact your DNS provider for help removing this.</p>

**Comment by patrickhars** [comment]
<p>Thank you for your reply <a class="mention" href="/thomasshaped">@thomasshaped</a>, when I deleted that IP, greentree.group is directed to <a href="http://patrickhars.github.io" rel="noopener nofollow ugc">patrickhars.github.io</a> instead of <a href="http://patrickhars.github.io/GREENTREE" rel="noopener nofollow ugc">patrickhars.github.io/GREENTREE</a>. Is there a way so greentree.group can directly direct to the page I wanted?</p>

**Comment by otter-computer** [comment, marked as answer]
<p>Hmm, taking a look inside your repository it seems like your site is inside a directory called <code>GREENTREE</code>:</p>
<aside class="onebox githubfolder">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">

      <a href="https://github.com/patrickhars/patrickhars.github.io/tree/main/GREENTREE" target="_blank" rel="noopener">github.com</a>
  </header>

  <article class="onebox-body">
    <h3><a href="https://github.com/patrickhars/patrickhars.github.io/tree/main/GREENTREE" target="_blank" rel="noopener">patrickhars/patrickhars.github.io</a></h3>

  <p><a href="https://github.com/patrickhars/patrickhars.github.io/tree/main/GREENTREE" target="_blank" rel="noopener">main/GREENTREE</a></p>

  <p><span class="label1">Contribute to patrickhars/patrickhars.github.io development by creating an account on GitHub.</span></p>

  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

<p>If you want your site to load directly at <a href="https://greentree.group/">https://greentree.group/</a> then you’ll need to move your site’s content outside of this folder to the root of your repository. The best way to do this would be to clone your repository locally, move your files outside of this <code>GREENTREE</code> folder, and push the changes back to GitHub.</p>

**Comment by patrickhars** [comment]
<p>Thank you for the solution, never thought it would be so simple :sweat_smile:</p>
